priv/templates/bin

```diff
@@ -69,16 +69,25 @@ ERTS_LIB_DIR="$ERTS_DIR/../lib"
 [ -f "$REL_DIR/$REL_NAME.boot" ] && BOOTFILE="$REL_NAME" || BOOTFILE=start
 cd "$ROOTDIR"
 
+{{! Define alternative field separator using ASCII US (unit separator). US is
+    not expected to appear in any argument. }}
+IFS_NORM="$IFS"
+IFS_ARGS="$(printf '\x1f')"
+
 # Save extra arguments
-ARGS=$*
+{{! Join extra arguments into a US-delimited string. }}
+IFS="$IFS_ARGS"
+ARGS="$*"
+IFS="$IFS_NORM"
```

Comment:
I believe that technically, `IFS` does not need to be restored for the rest of this file to work, because there is no other instance of word splitting (via unquoted variable expansion) besides `$ARGS` going on.

However, leaving it as a non-standard value would probably lead to more confusion down the line, and so the common shell pattern of restoring `IFS` after every unconventional use is followed here.